cfpb/cfgov-refresh
# Fix for video sharing

Fix for video sharing

## Removals

- Removed `cfgov/jinja2/v1/_includes/macros/share.html` because it's no longer needed.

## Changes

- Modified `cfgov/jinja2/v1/_includes/macros/video-player.html` to use `social-media` molecule.  Added static properties necessary to share content via the `social-media` molecule. 

- Modified `cfgov/unprocessed/css/video-player.less` to reference `social-media` molecule. 

## Testing
- run `gulp styes`
- Visit `http://localhost:8000/about-us/the-bureau/` and use the share buttons. The share buttons should work as expected ( Facebook share button won't work on localhost ). 

## Screenshots
- Email
<img width="656" alt="screen shot 2017-01-23 at 4 27 13 pm" src="https://cloud.githubusercontent.com/assets/1696212/22223403/d618d3fe-e188-11e6-98e8-40fc938b60ed.png">

-  Twitter
<img width="789" alt="screen shot 2017-01-23 at 4 27 57 pm" src="https://cloud.githubusercontent.com/assets/1696212/22223421/ee1581aa-e188-11e6-90cb-be0acc81657b.png">

- LinkedIn
<img width="782" alt="screen shot 2017-01-23 at 4 28 33 pm" src="https://cloud.githubusercontent.com/assets/1696212/22223452/08ed0aac-e189-11e6-9272-ce015080ecef.png">


- Buttons
<img width="213" alt="screen shot 2017-01-23 at 4 30 20 pm" src="https://cloud.githubusercontent.com/assets/1696212/22223514/4fd5bea0-e189-11e6-8639-fefdf7db950b.png">

## Notes

- The old text states `SHARE THIS VIDEO` and the new text states `SHARE THIS`. 
- Added the LinkedIn share button.
- At some point we will need to convert the video-player into a atomic component.


## Checklist
